sample/validation_rules.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
what's this file for?

Reply:
wanted to bypass the syntax check for pytest sample but seems it fails. Will delete it.